datasets/acute-care/extract_trio_vcf.py

```diff
@@ -0,0 +1,60 @@
+import click
+import hail as hl
+import json
+import os
+
+from itertools import chain
+
+
+"""
+simple procedural script that is provided with command line inputs
+- the JSON dictionary representing "family_name_1: [list, of, internal, sample, IDs], "
+- the project/dataset name, used to identify the buckets assc. with the specific project
+
+arguments passing mediated with click, so minimal hard-coding required 
+"""
+
+
+@click.command()
+@click.option('--json-str', help='this is the formatted json string of family_sample_lookups', type=click.STRING)
+@click.option('--dataset', help="name of the dataset to use (gcp bucket names)", type=click.STRING)
+def main(json_str: str, dataset: str):
+
+    # parse the families dict from the input string, e.g. '{"fam1":["sam1","sam2"]}'
+    families_dict = json.loads(json_str)
+
+    gcp_test = f"gs://cpg-{dataset}-test"
+    gcp_main = f"gs://cpg-{dataset}-main"
+    gcp_mt_full = os.path.join(gcp_main, "mt", f"{dataset}.mt")
+
+    # collect all unique sample IDs for a single filter on the MT
+    all_samples = set(chain.from_iterable(families_dict.values()))
```

Comment:
I don't think this one works, as values() will be the list of lists, so the whole lot needs to be flattened rather than the lists being added to a set